Automattic/jetpack
# Add/plans ingestion

#### Changes proposed in this Pull Request:

* Adds redux actions and selectors for getting plans from wpcom to the frontend

#### Testing instructions:



* check out this branch
* At the bottom of `_inc/client/state/site/actions.js` add the line `window.fetchAvailablePlans = fetchAvailablePlans;`
* `yarn build` to update the client
* on the test site, in the console, run `fetchAvailablePlans()(dispatch);` to create the actions. You should see them in the redux dev tools with the action `JETPACK_SITE_PLANS_FETCH` followed by `JETPACK_SITE_PLANS_FETCH_RECEIVE` and the plans object.


#### Proposed changelog entry for your changes:
